src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java

```diff
@@ -47,60 +48,60 @@ public InsolvencyDeltaProcessor(ApiClientService apiClientService,
      * Process CHS Delta message.
      */
     public void processDelta(Message<ChsDelta> chsDelta) {
+        MessageHeaders headers = chsDelta.getHeaders();
+        final ChsDelta payload = chsDelta.getPayload();
+        final String logContext = payload.getContextId();
+        final Map<String, Object> logMap = new HashMap<>();
+        final String receivedTopic =
+                Objects.requireNonNull(headers.get(KafkaHeaders.RECEIVED_TOPIC)).toString();
+        final String partition =
+                Objects.requireNonNull(
+                        headers.get(KafkaHeaders.RECEIVED_PARTITION_ID)
+                ).toString();
+        final String offset =
+                Objects.requireNonNull(headers.get(KafkaHeaders.OFFSET)).toString();
+
+        ObjectMapper mapper = new ObjectMapper();
+        InsolvencyDelta insolvencyDelta = null;
         try {
-            MessageHeaders headers = chsDelta.getHeaders();
-            final ChsDelta payload = chsDelta.getPayload();
-            final String logContext = payload.getContextId();
-            final Map<String, Object> logMap = new HashMap<>();
-            final String receivedTopic =
-                    Objects.requireNonNull(headers.get(KafkaHeaders.RECEIVED_TOPIC)).toString();
-            final String partition =
-                    Objects.requireNonNull(
-                            headers.get(KafkaHeaders.RECEIVED_PARTITION_ID)
-                    ).toString();
-            final String offset =
-                    Objects.requireNonNull(headers.get(KafkaHeaders.OFFSET)).toString();
-
-            ObjectMapper mapper = new ObjectMapper();
-            InsolvencyDelta insolvencyDelta = mapper.readValue(payload.getData(),
+            insolvencyDelta = mapper.readValue(payload.getData(),
```

Comment:
Shouldn't de-serialisation failure here be a Non-Retryable exception as per story?